src/main/java/com/icthh/xm/gate/config/MicroserviceSecurityConfiguration.java

```diff
@@ -56,10 +56,10 @@ public void configure(HttpSecurity http) throws Exception {
             .headers()
             .frameOptions()
             .disable()
-        .and()
+            .and()
```

Comment:
in this class only code formation was changed, you should leave code formatting style from old version